### ISSUE TITLE
Passing test example (5f99a7) fails(softly) for unrelated test

### ISSUE THREAD

**Issue by cptvitamin** [act-rules/act-rules.github.io#563]
Reference: https://github.com/act-rules/act-rules.github.io/blob/1d7899bd85219c6661aa1600526663a8c922cb9b/_rules/aria-attr-valid-5f99a7.md#passed-example-1

While running our ruleset against the ACT Rules testcases, I noticed that one of the passing examples for [ARIA-ATTRIBUTE-VALID](https://act-rules.github.io/rules/5f99a7) generates a warning for an unrelated rule that validates aria-attribute types.

In the example:
`<article aria-atomic>This is a decription of something cool...</article>`

While aria-atomic is a valid aria-* attribute, I don't believe it should be used without an explicit true/false value.

[aria-atomic](https://www.w3.org/TR/wai-aria/#aria-atomic) requires a value type of true/false. In this example, it is being used as [a boolean attribute as referenced in HTML5](https://www.w3.org/TR/html5/infrastructure.html#sec-boolean-attributes). In HTML5, 
> The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value.
If the attribute is present, its value must either be the empty string or a value that is an ASCII case-insensitive match for the attribute’s canonical name, with no leading or trailing white space.
> A boolean attribute without a value assigned to it (e.g. checked) is implicitly equivalent to one that has the empty string assigned to it (i.e. checked=""). As a consequence, it represents the true value.
> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether. 

However, WAI-ARIA1.1 [does not have a boolean attribute type](https://www.w3.org/TR/wai-aria-1.1/#propcharacteristic_value).  It has true/false[/mixed,undefined]. [WAI-ARIA1.1](https://www.w3.org/TR/wai-aria-1.1/#typemapping) also contains a Note that says 
`The suggested mappings for true/false values in HTML 5 use Keyword and enumerated attributes with allowed values of true and false, instead of using the HTML 5 boolean value type.`

I interpret this as meaning that an explicit value should be specified on all aria-* attributes that require true/false[/mixed,undefined] value types and just the presence of the attribute does not signify a particular state. We've gone back and forth internally as to whether or not this condition should raise a failure or a warning. Nonetheless, I support an explicit value, especially since the "default value is false"  Which could easily lead someone to believe that `aria-atomic` by itself represents `aria-atomic=false`.

**Comment by Jym77** [comment]
I agree with your interpretation, especially on the default value.
[The description of `aria-atomic`](https://www.w3.org/TR/wai-aria-1.1/#aria-atomic) is quite explicit on that:
> When the content of a live region changes, user agents SHOULD examine the changed element and traverse the ancestors to find the first element with aria-atomic set, and apply the appropriate behavior for the cases below.
> 1. If none of the ancestors have explicitly set aria-atomic, the default is that aria-atomic is false, and assistive technologies will only present the changed node to the user.
> 2. If aria-atomic is explicitly set to false, assistive technologies will stop searching up the ancestor chain and present only the changed node to the user.
> 3. If aria-atomic is explicitly set to true, assistive technologies will present the entire contents of the element, including the author-defined live region label if one exists.

`aria-atomic` (with no value) has a slightly different meaning than `aria-atomic=false` (the former means "inherit" more than "false"), but it is clearly not the same as `aria-atomic=true`.

OTOH, I still understand that `aria-atomic` (with no value) is valid and thus the example is correct…

**Comment by jkodu** [comment]
A boolean value of `true` has been provided for the passing example 1 referred here. 

See - https://github.com/act-rules/act-rules.github.io/blob/develop/_rules/aria-attr-defined-5f99a7.md#passed-example-1
